soccer/planning/Path.hpp

```diff
@@ -18,6 +25,9 @@ class Path {
     Path() : _startTime(0) {}
     virtual ~Path() {}
 
+    /// Returns true if the path is non-empty
+    virtual bool valid() const { return true; }
```

Comment:
I mean its possible. But we should really never have a pathPlanner return an empty path.
We can just add checks in the pathplanner and return nullptr if something really irrecoverable happens.
 On another note, we could make Path stuff mostly immutable since we have the subpath and compositePath stuff setup.